neqo-server/src/main.rs

```diff
@@ -836,6 +836,15 @@ fn main() -> Result<(), io::Error> {
     init_db(args.db.clone());
 
     if let Some(testcase) = args.qns_test.as_ref() {
+        assert!(
```

Comment:
I'd rather leave the footgun in place (that what these tools are for) and only set the version if it is unset.